api/fossa/locator.go

```diff
@@ -82,9 +84,26 @@ func (l Locator) URL() string {
 	if err != nil {
 		log.Fatalf("Invalid FOSSA URL: %s", err.Error())
 	}
+
+	if org.UsesSAML {
+		url = SamlURL(*url, org)
+	}
+
 	return server.ResolveReference(url).String()
 }
 
+// SamlURL constructs a SAML redirect URL for a given FOSSA URL
+func SamlURL(uri url.URL, org Organization) *url.URL {
+	samlBase, err := url.Parse("/account/saml/" + strconv.Itoa(org.OrganizationID))
+	if err != nil {
+		log.Fatalf("Invalid SAML URL construction: %s", err.Error())
```

Comment:
I think this should be `log.Warnf` to follow line 70. The project is still uploaded at this point if we get an error correct? If so I think we should make this a warning because `analyze` technically succeeded

Reply:
Technically, it doesn't matter, since this can't fail.  It's parsing a constant value and any integer.  But I can change it.

Reply:
Thats a good point... this is going to be nitpicky but can we change it `log.Warn` to follow our convention of not logging fatal unless absolutely necessary?